torch/overrides.py

```diff
@@ -185,6 +185,7 @@ def get_ignored_functions() -> Set[Callable]:
         torch.sym_max,
         torch.sym_min,
         torch.sym_not,
+        torch.is_concrete_int,
```

Comment:
I would NOT expose this from top-level torch.* namespace

Reply:
Sounds good